src/mlpack/methods/kde/kde.hpp

```diff
@@ -257,6 +352,41 @@ class KDE
 } // namespace kde
 } // namespace mlpack
 
+//! Set the serialization version of the KDE class.
+/* TODO FIX
```

Comment:
I don't know if `TODO FIX` is needed---actually, I think this is not fixable due to the limitations of C macros.  The comment is still useful though. :)